civiform_config.example.sh

```diff
@@ -115,6 +115,11 @@ export STAGING_HOSTNAME=""
 # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 export CIVIFORM_TIME_ZONE_ID="America/Los_Angeles"
 
+# OPTIONAL
+# If enabled, allows exporting Prometheus server metrics over HTTP at "/metrics"
+# Defaults to false.
+# export CIVIFORM_SERVER_METRICS_ENABLED=false
```

Comment:
doesn't need to be commented out.

Reply:
It doesn't, but the thought was that by leaving it commented out it would make it clear that it doesn't need to be set and will be false, otherwise "default to false" could be interpreted as the template sets it to false before getting customized/edited